datasets/esa-worldcover.yaml

```diff
@@ -29,25 +30,48 @@ Resources:
     Region: eu-central-1
     Type: S3 Bucket
     RequesterPays: False
+    Explore:
+    - '[STAC endpoint](https://services.terrascope.be/stac/)'
```

Comment:
I noticed this didn't work with STAC Browser due to missing headers: https://radiantearth.github.io/stac-browser/#/external/services.terrascope.be/stac/

This doc may have some useful info: https://www.stackhawk.com/blog/fixing-no-access-control-allow-origin-header-present/.

Reply:
Hi Mike, thanks. We need to look into that. Right now it's using our terrascope.be STAC API, maybe that's not fully compliant, we'll need to see how to update that, but we have a notebook that uses it.
The github repo is now public.